docker-compose.prod.yml

```diff
@@ -3,8 +3,12 @@ services:
   production:
```

Comment:
Note that strictly the `docker-compose` isn't used in production but you can use it to validate the environment before deployment to wheatley (we use Kubernetes to manage the cluster)

Reply:
Ah. Good to know. Looking into the deployment environment is what I'm doing next. Thanks for the heads up.